static/js/admin/vue-permissions-editor/src/router.js

```diff
@@ -0,0 +1,17 @@
+import Vue from 'vue'
+import Router from 'vue-router'
+import modalEditor from "./components/modalEditor";
+
+Vue.use(Router)
+
+export default new Router({
+  mode: 'history',
+  base: process.env.BASE_URL,
+  routes: [
+    {
+      path: '/admin/list/:uuid?',
```

Comment:
This is causing the search page javascript to fail since it is at path `/admin/search/:uuid`, should work for that too